File: git-function-history-lib/src/test_functions.go
package main

import (
	"fmt"
	"os"
)

func main() {
	empty_test("1", 2, "3")
	fmt.Println("Hello World!")
}
// doc comment

func empty_test(c, a int, b string) {
	fmt.Println("Hello World!")
}

File: git-function-history-lib/src/test_functions.c
#include <stdio.h>

void test_function(void);

static void test_function2(void)
{
    printf("Hello World!");

    // printf("Hello World!" );
}

int main()
{
    printf("Hello World!");
    test_function();
    test_function2();
    // test_functions();
    // test_functions2();
    return 0;
}

void test_function(void)
{
    printf("Hello World!");
}

int empty_test()
{f}

